hr_timesheet_sheet/tests/test_hr_timesheet_sheet.py

```diff
@@ -398,12 +402,11 @@ def test_6(self):
         self.assertEqual(len(timesheet_3_4_and_5), 3)
 
         timesheet_6 = self.aal_model.create({
-            'name': 'z',
+            'name': empty_name,
             'project_id': self.project_1.id,
             'employee_id': self.employee.id,
             'unit_amount': 2.0,
         })
-        timesheet_5.name = '/'
```

Comment:
put empty_name here

Reply:
Since this line was missing at all in v12